### ISSUE TITLE
[tree view] All selectable items should have aria-selected set to false when they are not selected.

### ISSUE THREAD

**Issue by joserodolfofreitas** [mui/mui-x#14982]
### Summary

All selectable items must have aria-selected=“false” by default (then aria-selected=“true” when selected). For the moment, we only have the aria-selected=“true” when it's selected, but without the aria-selected=“false”.

Accessibility issue reported by iValua.

**Search keywords**:

**Comment by oliviertassinari** [comment]
Reference for why this report from iValue is valid (it's frequent to receive invalid a11y reports)

> All nodes that are selectable but not selected have either aria-selected or aria-checked set to false.

https://www.w3.org/WAI/ARIA/apg/patterns/treeview/

**Comment by abdermaiza** [comment]
@joserodolfofreitas is this issue is closed because not considered as an issue, or already fixed?...

**Comment by joserodolfofreitas** [comment]
@abdermaiza, it's closed as completed.
The implementation is at: https://github.com/mui/mui-x/pull/14991

**Comment by abdermaiza** [comment]
Can we test it somehow?

**Comment by oliviertassinari** [comment]
@abdermaiza You can the preview link that are on the PR descriptions: #14991.
